### DISCUSSION TITLE
How do I get a 5/6th window shortcut?

### DISCUSSION THREAD

**Discussion by chris-brinkman** [rxhanson/RectanglePro-Community#193] [Q&A] [answered]
I like to use the 1/6th shortcut to place a small, 1/6th size, Chrome window to the right side of my screen, typically for YouTube. I'd like a shortcut for the remaining 5/6th for some other window. Thank you.

**Comment by rxhanson** [comment, marked as answer]
Here it is in Custom Shortcuts:

![image](https://user-images.githubusercontent.com/13651296/208218033-773446ce-ec11-4820-9bee-2821b56b911d.png)

**Reply by chris-brinkman** [reply]
Thank you!
